src/CUBLAS.jl

```diff
@@ -11,8 +11,11 @@ module CUBLAS
 
 importall Base.LinAlg.BLAS
 
-using CUDArt
-using CUDArt.rt.cudaStream_t
+using CUDAdrv: OwnedPtr, CuArray, CuVector, CuMatrix
+
+CuVecOrMat{T} = Union{CuVector{T},CuMatrix{T}}
+
+const cudaStream_t = Ptr{Void}
```

Comment:
why do you need this? there's `CuStream` with an `unsafe_convert` (ie. can be passed to ccall)